app/channel.go

```diff
@@ -2904,7 +2904,7 @@ func (a *App) SearchChannelsUserNotIn(teamID string, userID string, term string)
 func (a *App) MarkChannelsAsViewed(channelIDs []string, userID string, currentSessionId string, collapsedThreadsSupported bool) (map[string]int64, *model.AppError) {
 	// I start looking for channels with notifications before I mark it as read, to clear the push notifications if needed
 	channelsToClearPushNotifications := []string{}
-	if *a.Config().EmailSettings.SendPushNotifications {
+	if a.canSendPushNotifications() {
```

Comment:
There is a minor behaviour change here. Before, we were just checking the "SendPushNotifications" setting, so it was not checking the license. Now, in case the license does not allow it, it will not send these notifications. I think the behaviour is expected, but still wanted to point that out.

Reply:
Maybe good to add this in the release notes.